### PR TITLE
Individual and Wildcard certificate

### PR DESCRIPTION
**Pull request by jenting**
# Describe your changes

Describes what is the individual certificate and the wildcard certificate.

- For subdomains: add both the individual certificate and the wildcard certificate
- For subpaths: add the individual certificate since it only has one domain name

# Related Issues / Projects
Please provide links to Bugzilla and other GitHub projects with your description if they are related to the changes.

Fixes: #835

### CODE REVIEW

File: adoc/admin-monitoring-stack.adoc [hunk 1]
```diff
@@ -198,6 +104,63 @@ In order to provide additional security by using TLS certificates, please make s
...
+----
+
+==== TLS Certificate
+
+You must configure your certificates for the components as secrets in the {kube} cluster.
+Get certificates from your certificate authority.
+
+. Individual certificate
++
+Single-name TLS certificate protects a single sub-domain, and it means each sub-domain owns its private key. From the security perspective, it is recommended to use a individual certificate. However, you have to manages the private key and the certificate rotation separately.
```
Comment (innobead) [inline adoc/admin-monitoring-stack.adoc:139]:
```suggestion
Single-name TLS certificate protects a single sub-domain, and it means each sub-domain owns its private key. From the security perspective, it is recommended to use individual certificates. However, you have to manage the private key and the certificate rotation separately.
```

File: adoc/admin-monitoring-stack.adoc [hunk 2]
```diff
@@ -198,6 +104,63 @@ In order to provide additional security by using TLS certificates, please make s
...
++
+.Note Down Secret Names For Configuration
+[IMPORTANT]
+====
+Should you choose to secure each service with an individual certificate, you must repeat the step below for each component and adjust the name for the individual secret each time. Please note down the names of the secrets you have created.
```
Comment (innobead) [inline adoc/admin-monitoring-stack.adoc:144]:
```suggestion
When you choose to secure each service with an individual certificate, you must repeat the step below for each component and adjust the name for the individual secret each time. Please note down the names of the secrets you have created.
```

File: adoc/admin-monitoring-stack.adoc [hunk 3]
```diff
@@ -198,6 +104,63 @@ In order to provide additional security by using TLS certificates, please make s
...
+
+In this example, the secret name is `monitoring-tls`.
+====
+
+. Wildcard certificate
++
+Wildcard TLS allows you to secure multiple sub-domains with one certificate and it means multiple sub-domains share the same private key. However, you could add more sub-domains without having to redeploy the certificate and more, saves the certificate costs.
```
Comment (innobead) [inline adoc/admin-monitoring-stack.adoc:151]:
```suggestion
Wildcard TLS allows you to secure multiple sub-domains with one certificate and it means multiple sub-domains share the same private key. So, you could add more sub-domains without having to redeploy the certificate and more, saves the certificate costs.
```

**Comment by jenting** [comment]
CI is not happy with the un-related change.

**Comment by jenting** [comment]
okay, address the format problem to make CI happy.

**Pull request merged**